src/software/jetson_nano/ansible/playbooks/remote_flash.yml

```diff
@@ -77,11 +87,16 @@
 
       # Rescue section only runs if any task in this block fails
       rescue:
-        - name: Error Recovery - Ensure Announcement Service is Running
+        - name: Error Recovery - Ensure Announcement Services are Running
           become: true
           become_method: sudo
           ansible.builtin.systemd:
-            name: announcement
+            name: wifi_announcement
+            masked: no
+            daemon_reload: yes
+            state: started
+          ansible.builtin.systemd:
+            name: ethernet_announcement
```

Comment:
can you update the ansible README to include the name of the names of these services. Specifically, the line that says:

Possible service tags: thunderloop, announcement, display. Possible action tags: stop, sync, start 

should become 

Possible service tags: thunderloop, wifi_announcement, ethernet_announcement,  display, Possible action tags: stop, sync, start

Reply:
fixed